client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/problem/rpc/MockDatabaseDto.java

```diff
@@ -0,0 +1,54 @@
+package org.evomaster.client.java.controller.api.dto.problem.rpc;
+
+/**
+ * mock object for database
+ */
+public class MockDatabaseDto {
```

Comment:
add bit more info here on what Mocking a Database is here.
also, is it something specific only to RPC? or could it applied to REST and GraphQL as well? if so, shouldn't be under `.../rpc`. if not, add bit mention on why